packages/vm/src/conversion.rs

```diff
@@ -0,0 +1,45 @@
+use std::any::type_name;
+use std::convert::TryFrom;
+
+use crate::errors::{ConversionErr, Result};
+
+/// Safely converts input of type FromType to u32.
```

Comment:
I assume you wanted to make this more generic? Either change the doc or the code.

I think this would work:

`pub fn to_u32<T: TryInto<u32>(input: T) -> Result<u32>` ...

Reply:
Ahhhhaha! Thanks! I gave up at some point because I could not get it to work with TryFrom. But TryInto works much better. I successfully tested it for usize, u64 and i32 now :)